src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -416,19 +480,26 @@ class ThumbnailsPanel extends React.PureComponent {
           }}
         >
           {({ measureRef }) => (
-            <div ref={measureRef} className="virtualized-thumbnails-container" >
+            <div ref={measureRef} className="virtualized-thumbnails-container"
+              style={{
+                // when 'shouldShowControls' is true but documentControlHeight isn't set yet, add a maxHeight to keep the height from re-measuring
+                maxHeight: shouldShowControls && !documentControlHeight ? height : null,
```

Comment:
Without the height I was getting the scrollbar behind the `documentControl` component 

![thumbnail_Issue](https://user-images.githubusercontent.com/45575633/72496616-f885b100-37de-11ea-963c-5a2a55b220be.gif)

The height is mainly to keep that from happening (will be bad if there was 500+ pages the handle might get covered ). With the way the virtualized-list is working I wasn't sure if there a better way to reduce the height of it  when the `documentControl` is displayed

Reply:
If the `documentControl` is absolutely positioned, then I don't think we need to update the height of the list. But as you said it will make it on top of the scroll bar. So I think what we have is good for now unless we can find some other ways to make the code look nicer haha